installing/installing_aws/installing-aws-vpc.adoc

```diff
@@ -0,0 +1,61 @@
+[id="installing-aws-vpc"]
+= Installing a cluster on AWS to an existing VPC
+include::modules/common-attributes.adoc[]
+:context: installing-aws-vpc
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster into an existing VPC on
```

Comment:
You never spell out VPC anywhere. I'd suggest spelling out at least once at the beginning of the assembly.